spec/system/add_search_rspec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Search function', type: :system do
+  before do
+    @ruby_programming_language = create :programming_language, name: 'Ruby'
+    create :programming_language, name: 'Javascript'
+    create :programming_language, name: 'Php'
+
+    @ruby_devloper = create :developer, email: 'phan.nguyen@example.com'
+    @ruby_devloper.developer_programming_languages.create(
+      programming_language: @ruby_programming_language
+    )
+  end
+
+  it 'Being able to search developers who can write Ruby' do
```

Comment:
The description sounds weird.
First of all, `Being` should be a lowercase because the sentence starts with `it`.
I would write `shows matched records when a programming language is specified(or chosen)`.
Ruby or not is not important.
You write a description that explains the logic.

Reply:
@jp-ryuji :kissing: